cdc/cdc.cc

```diff
@@ -62,7 +62,7 @@ template<> struct hash<std::pair<net::inet_address, unsigned int>> {
 
 using namespace std::chrono_literals;
```

Comment:
Commitlog: some info about what this is doing

Reply:
The patch is adding hundreds of lines of codes, and leaves it to the reviewer to reverse-engineer.

Reply:
Done